varats-core/varats/project/project_util.py

```diff
@@ -197,14 +197,18 @@ def __init__(
 
     @property
     def name(self) -> str:
+        """Name of the Binary."""
```

Comment:
You've written it small in all other places.

```suggestion
        """Name of the binary."""
```